README.md

```diff
@@ -58,15 +58,83 @@ For more information, read the [documentation](https://thelounge.github.io/docs/
 
 ## Development setup
 
-To run the app from source, just clone the code and run this in your terminal:
+
+
+1. Fork lounge to your github account from the top right corner.
+ 
+2. Clone from your github account.
+
+    ```
+    git clone https://github.com/<username>/lounge/
+    ```
+
+3. Install dependencies, it takes a while
+
+    ```
+    cd lounge/
+    npm install
+    ``` 
+
+4. Run lounge to generate configuration files
+
+    ```
+    npm start
+    ```
+
+5. Create new branch and switch into it, name should contain lounge issue ticket 
+    number, create one in https://github.com/thelounge/lounge/issues if missing
+
+    ```
+    git checkout -b XXX-feature-name
+    ``` 
+
+6. Edit code to add your feature.
+
+7. Build and test when its needed. You will have to run `npm run build` if you change or add anything in `client/js/libs` or `client/views`.
+ 
+    ```
+    npm run build
+    npm run test
+    ```
+
+8. If test is passed, add changes and make commit
+
+    ```
+    git add -i 
+    git commit -m "#xxx - feature info"
+    ```
```

Comment:
This is also git stuff and not The Lounge stuff, so it doesn't belong to our README, I think.